rke/cis_config.go

```diff
@@ -4,29 +4,74 @@ import (
 	v3 "github.com/rancher/types/apis/management.cattle.io/v3"
 )
 
+const (
+	reasonNoConfigFile = `Cluster provisioned by RKE doesn't require or maintain a configuration file for kube-apiserver.
+All configuration is passed in as arguments at container run time.`
+	reasonForPSP       = `Enabling Pod Security Policy can cause issues with many helm chart installations`
```

Comment:
Both the PSP and NetPol reason seems to be very short and will raise a lot of questions. Expanding this would be beneficial for users, not sure what the context is of this message but I assume it's our reason why the cluster does not have PSP enabled or Network Policies defined.

Reply:
We need to revisit the reasons for sure. The list is also not exhaustive yet, wanted to get the code merged in first.